spring-boot-autoconfigure/src/main/java/org/axonframework/springboot/autoconfig/JpaAutoConfiguration.java

```diff
@@ -46,7 +46,8 @@
 @ConditionalOnBean(EntityManagerFactory.class)
 @RegisterDefaultEntities(packages = {
         "org.axonframework.eventhandling.tokenstore",
-        "org.axonframework.modelling.saga.repository.jpa"
+        "org.axonframework.eventhandling.deadletter.jpa",
```

Comment:
Guess I should've resolved the previous comment just yet. It contains the following point:

> From another perspective you could however state "where not auto wiring the JPA DLQ, so why register the entities"?